specification/sql/resource-manager/readme.md

```diff
@@ -292,6 +292,17 @@ input-file:
 - Microsoft.Sql/preview/2017-03-01-preview/serverDnsAliases.json
 - Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json
 
+### Tag: schema-2017-03-preview

```

Comment:
Please delete this and add to `package-pure-2017-03-preview` instead

Reply:
Done.